doc/api/cli.md

```diff
@@ -359,6 +359,21 @@ Error: Access to this API has been restricted
 }
 ```
 
+### `--async-context-frame`
```

Comment:
Shouldn't it start with `experimental`? `--experimental-async-context-frame`. I mean, I don't recall having such a policy, but it seems a convention for experimental behaviours.

Reply:
Switched it to using the prefix. I wasn't thinking of it being flagged so much to be experimental as to just not break Electron, but I suppose it _does_ make sense to call this experimental.